.github/workflows/linting.yml

```diff
@@ -19,4 +19,4 @@ jobs:
     - name: Run flake8
       run: |
         flake8 easybuild/tools
- 
\ No newline at end of file
+        flake8 test/
```

Comment:
Please do not change this during the sprint. My suggestion is to enable this at the very end for `.`. Otherwise all other PRs will conflict when changing this too.
Oh and according to GH it is missing the final newline and the trailing slash is not required :)

Reply:
Sure if you want to make those dependent PRs but IMO it is easier to verify locally via flake8 and don't change this yet.

Reply:
For `easyblocks` repo, I definitely agree we should hold off touching (actually adding) `linting.yml` until we're done.